<launch>
  <node pkg="laser_to_pointcloud" type="laser_to_pointcloud"
      name="laser_to_pointcloud">
  </node>

  <node pkg="cluster_pointcloud" type="cluster_pointcloud"
      name="cluster_pointcloud">
  </node>

  <node pkg="cluster_pointcloud" type="cluster_pointcloud"
      name="find_truck">
  </node>
</launch>
